src/app/public/modules/key-info/key-info.component.scss

```diff
@@ -18,3 +19,15 @@
     }
   }
 }
+
+@include sky-theme-modern {
+  .sky-key-info {
+    &.sky-key-info-horizontal {
+      padding: 0 $sky-theme-modern-space-md 0 0;
```

Comment:
We should not build this spacing into the component, it will create difficulty in achieving correct spacing between elements. Consumers should use the appropriate spacing classes/techniques to achieve results per spacing guidelines.

Reply:
Removed.